src/platyPS/platyPS.psm1

```diff
@@ -1777,3 +1777,19 @@ function Convert-PsObjectsToMamlModel
     return $MamlCommandObject
 }
 #endregion
+
+#region Parameter Auto Completers
+# Completers are storted in .\psplaty.AutoCompleters.ps1
+
+# Add all completers in this IF block to ensure TabExpansionPlusPlus is installed, else there is no -Description Param in the V5 Register-ArgumentCompleter Command.
+if (Get-Command -Name Register-ArgumentCompleter -Module TabExpansionPlusPlus -ErrorAction Ignore) {
```

Comment:
There are two ways how `Register-ArgumentCompleter` could be in available: with TabExpansionPlusPlus or with PS v5 (Microsoft.PowerShell.Core). We are fine with ever one of them. Le's just resolve it by name and put comment about it.